egs/vctk/TTS/prepare.sh

```
@@ -44,8 +50,18 @@ if [ $stage -le 0 ] && [ $stop_stage -ge 0 ]; then
   #
   #   ln -sfv /path/to/VCTK $dl_dir/VCTK
   #
+  # Alternatively, use --local-data-dir to point to an existing VCTK directory:
+  #
+  #   bash prepare.sh --local-data-dir /path/to/VCTK
+  #
   if [ ! -d $dl_dir/VCTK ]; then
-    lhotse download vctk --use-edinburgh-vctk-url ${use_edinburgh_vctk_url} $dl_dir
+    if [ -n "$local_data_dir" ]; then
+      log "Using local data directory: $local_data_dir"
+      mkdir -p $dl_dir
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)